internal/idutils/doc.go

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Package idutils provides a set of helper functions to convert ids.
+
+Functions in big_endian_converter.go help converting uint64 ids to TraceID
+and SpanID using big endian, and vice versa.
+*/
```

Comment:
This should use the line comment style:

```suggestion
// Package idutils provides a set of helper functions to convert ids.
//
// Functions in big_endian_converter.go help converting uint64 ids to TraceID
// and SpanID using big endian, and vice versa.
```